### ISSUE TITLE
Running multiple thread with pm2?

### ISSUE THREAD

**Issue by elmondogri** [wagenaartje/neataptic#116]
Hello, I am new to ML and even to Node.js,

Would it be faster and is is possible to run neataptic under pm2 cluster mode?

Else what would be the fastest way to run lstm(50,6,3) training with 2500 data, at the moment my caluclation shows around 180 days to train...

thank you

**Comment by lokidaibel** [comment]
No. It will just spawn multiple processes. You need to adjust your code.
